haproxy/README.md

```diff
@@ -18,7 +18,19 @@ The Haproxy check is included in the [Datadog Agent][2] package, so you don't ne
 
 #### Prepare HAProxy
 
-##### Versions < 2
+##### Using Prometheus
+
+The check supports a newer implementation starting with HAProxy version 2 (enterprise version 1.9rc1) that is based on a Prometheus endpoint:
+
+1. Configure your `haproxy.conf` using the [official guide][16].
+
+2. [Enable](#configuration) the setting `use_prometheus` in `haproxy.d/conf.yaml`.
+
+3. [Restart HAProxy to enable the stats endpoint][3].
+
+Note: The prometheus endpoint can also be set in prior version using the [HAProxy exporter][17]
```

Comment:
This is a bit confusing. The first sentence in this section says "this applies 2.0+ or Enterprise 1.9.1rc1+", but here we say "… but if you're using an older version, this can also work if you install the exporter yourself".